x-pack/platform/plugins/shared/cases/server/connectors/cases/index.ts

```diff
@@ -82,18 +79,19 @@ export const getCasesConnectorType = ({
       throw new Error('Cannot authorize cases. Owner is not defined in the subActionParams.');
     }
 
-    const owner = isServerlessSecurity
-      ? SECURITY_SOLUTION_OWNER
+    const owner = serverlessProjectType
```

Comment:
Here we assign the project type, which may not be the same as the owner's. Should we use the same logic as in `x-pack/platform/plugins/shared/cases/common/utils/owner.ts`?